docs/customization/reports.md

```diff
@@ -82,6 +84,8 @@ class DeviceConnectionsReport(Report):
 
 As you can see, reports are completely customizable. Validation logic can be as simple or as complex as needed. Also note that the `description` attribute support markdown syntax. It will be rendered in the report list page.
 
+In the same way scripts do, reports run from within Netbox's own environment and can access the objects inside Netbox directly. The [Netbox Shell](../administration/netbox-shell.md) is a good resource to keep in hand, since it allows to see the objects in Netbox in the same way the test methods of a report do.
```

Comment:
Please correct "Netbox" to NetBox. This should also probably render as a tip admonition.

Reply:
Done